app/controllers/admin/reports_controller.rb

```diff
@@ -46,6 +46,7 @@ def assign_view_data
 
       # Initialize data
       params[:display_summary_row] = true if request.get?
+      params[:fields_to_show] = @report.columns.keys if request.get?
```

Comment:
It would be much cleaner to use `params_fields_to_show` and leave the params unaltered. Never a good idea to change the `params` object.

Reply:
Ok, see commit: #560c1e2e0:

```ruby
@params_fields_to_show = if request.get?
                           @report.columns.keys
                         else
                           params[:fields_to_show]
                         end
```